### ISSUE TITLE
Should count CRLF as one character

### ISSUE THREAD

**Issue by dkent600** [protofire/solhint#109]
solhint 1.5.1 counts CRLF as two characters.  Should only be one.

**Comment by fvictorio** [comment]
Hi @dkent600. Could you give us an example where this is a problem?

**Comment by dkent600** [comment]
@fvictorio  It happens for me with lines that are 120 characters long (right at the limit), which solhint thinks are 121.

Example (including the preceding spaces):

```
    function executeProposal(bytes32 _proposalId, int256 _param) external onlyVotingMachine(_proposalId) returns(bool) {
```

**Comment by fvictorio** [comment]
I see, thanks!

**Comment by fvictorio** [comment]
Hey @dkent600, sorry for taking so long to handle this. It should be fixed in version 2.2.0, in case you want to test it. Thanks again for the report!
